client/src/test/java/io/split/engine/common/SyncManagerTest.java

```diff
@@ -29,9 +32,14 @@ public void startWithStreamingFalseShouldStartPolling() throws InterruptedExcept
         TelemetryStorage telemetryStorage = Mockito.mock(TelemetryStorage.class);
         _gates.sdkInternalReady();
         TelemetrySynchronizer telemetrySynchronizer = Mockito.mock(TelemetrySynchronizer.class);
+        ImpressionsManagerImpl impressionsManager = Mockito.mock(ImpressionsManagerImpl.class);
```

Comment:
Nit. here; if `SyncManagerImp` expects an `ImpressionsManager` (the interface) this could be
```suggestion
        ImpressionsManager impressionsManager = Mockito.mock(ImpressionsManager.class);
```

Reply:
I replaced it.